ext/src/js/modules/main.js

```diff
@@ -16,200 +16,206 @@ import {
+
+async function createBookmark(e) {
+    e.preventDefault();
+    form = new FormData(document.forms['add']);
+    updateBoundVar([{ 'onafterdownload': true }, { 'onmain': false }]);
+    renderProgressBar(document.getElementById('omnom_status'));
+    createSnapshot();
+}
+async function saveBookmark(snapshotData) {
+    form.set('tags', tagInput.getTags().join(','));
+
+    //console.time('createSnapshot');
+    //console.timeEnd('createSnapshot');
+    if (isDebug()) {
+        debugPopup(snapshotData['dom']);
+        return;
     }
-
-    /* ---------------------------------*
-     * Save bookmarks                   *
-     * ---------------------------------*/
-
-    async function saveBookmark(e) {
-        e.preventDefault();
-        const form = new FormData(document.forms['add']);
-        form.set('tags', tagInput.getTags().join(','));
-        updateBoundVar([{ 'onafterdownload': true }, { 'onmain': false }]);
-        renderProgressBar(document.getElementById('omnom_status'));
-
-        console.time('createSnapshot');
-        const snapshotData = await createSnapshot();
-        console.timeEnd('createSnapshot');
-        if (isDebug()) {
-            debugPopup(snapshotData['dom']);
-            return;
-        }
-        const snapshotBlob = new Blob([snapshotData['dom']], { type: 'text/html' });
-        form.append('snapshot', snapshotBlob);
-        form.append('snapshot_text', snapshotData['text']);
-        form.set('favicon', snapshotData['favicon']);
-        const requestBody = {
-            method: 'POST',
-            body: form,
-            // headers: {
-            //     'Content-type': 'application/json; charset=UTF-8'
-            // }
-        }
-        await fetch(`${getOmnomUrl()}add_bookmark`, requestBody)
-            .then((resp) => checkStatus(resp)).then(() => {
-                destroyProgressBar();
-                renderSuccess('Snapshot successfully saved!');
-            }, (err) => {
-                destroyProgressBar();
-                renderError(`Failed to save bookmark! Error: ${err.statusText}`, err);
+    const snapshotBlob = new Blob([snapshotData['dom']], { type: 'text/html' });
+    form.append('snapshot', snapshotBlob);
+    form.append('snapshot_text', snapshotData['text']);
+    form.set('favicon', snapshotData['favicon']);
+    const requestBody = {
+        method: 'POST',
+        body: form,
+        // headers: {
+        //     'Content-type': 'application/json; charset=UTF-8'
+        // }
+    };
+    await fetch(`${getOmnomUrl()}add_bookmark`, requestBody)
+        .then((resp) => checkStatus(resp)).then(() => {
+            destroyProgressBar();
+            renderSuccess('Snapshot successfully saved!');
+        }, (err) => {
+            destroyProgressBar();
+            renderError(`Failed to save bookmark! Error: ${err.statusText}`, err);
+        });
+}
+
+/* ---------------------------------*
+    * Template managemen               *
```

Comment:
Can you fix the comment for management? 😄  I always forget it, and it's bugging the hell out of me.